app/src/main/res/values/strings.xml

```diff
@@ -745,5 +745,7 @@ You only need to do this once, until the next time you select a new location for
 
     <string name="quick_desc">Files recently accessed through Amaze</string>
     <string name="recent_desc">Recently created or modified files</string>
+    <string name="error_cannot_find_way_open">Could not find way to open requested file</string>
+    <string name="error_cant_decompress_that_file">Can\'t extract or show that archive</string>
```

Comment:
`that archive` can be replaced with `the requested archive` IMO

Reply:
Is `the archive` ok? `requested` seems unnecessarily long

Reply:
> Is `the archive` ok? `requested` seems unnecessarily long

How about using `Can't process the requested archive` and removing `extract or show`?

Reply:
Fixed!